enhancements/installer/cloud-custom-dns.md

```diff
@@ -0,0 +1,617 @@
+---
+title: cloud-custom-dns
+authors:
+  - "@sadasu"
+reviewers: 
+  - "@patrickdillon, for Installer aspects"
+  - "@Miciah, for Ingress Controller aspects"
+  - "@cybertron, for on-prem networking aspects"
+  - "@2uasimojo, for hive expertise"
+approvers:
+  - "@zaneb"
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - "@joelspeed"
+creation-date: 2022-10-25
+last-updated: 2023-01-23
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - "https://issues.redhat.com/browse/CORS-1874"
+---
+
+# Custom DNS for AWS
```

Comment:
Is this enhancement just for AWS, or for AWS+GCP+Azure as implied by PR title and below Summary?

Reply:
@2uasimojo for AWS, GCP and Azure. Fixed the title of this section that is causing confusion.